methods/import_media_excel_file/display.yaml

```diff
@@ -14,8 +14,9 @@ method-suggestions :
 parameters :
     inputFile :
         ui-name : Name of EXCEL media file
-        short-hint : Select a media file in EXCEL format
-        long-hint  : Select a media file in EXCEL format
+        short-hint : Select a media file in Excel format
+        long-hint  : |
+        Media file should have 4 tab-separated fields: compounds, concentrations, minflux, maxflux. Filename should end with ".xls" or ".xlsx"
```

Comment:
Although syntax tests pass, I think this might not be parsing as expected.  I think because the line starts at the same indent level as long hint, it is interpreting "Media file should have 4 tab-separated fields" as a new key...  That seems to be what happens in the http://www.yamllint.com parse.  Can you add the indent, or try this in CI first before we pull this in?  Thanks.